### PR TITLE
pyrunner:0.1.0

### PR DESCRIPTION
**Pull request by peng1999**
I am submitting
- [x] a new package
- [ ] an update for a package

Description: Run Python code in Typst.

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name that isn't the most obvious or canonical name for what the package does
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] tested my package locally on my system and it worked
- [x] `exclude`d PDFs or README images, if any, but not the LICENSE

### CODE REVIEW

**Comment by Andrew15-5** [comment]
In the future, please consider using IEC units instead of the deprecated SI units for measuring size of information. They are confusing, because they should show multiples of 10, but a lot of people use them as multiples of 2, but multiples of 10 as well. I had to see the source of 11.94 MiB to make sure that you are using SI units correctly (a lot of people and software don't, including Microsoft, Google, Yandex and others). See https://en.wikipedia.org/wiki/Binary_prefix.

<details><summary>Deprecation quote for the wiki page</summary>
<p>

> Note: in information technology, especially for measurements of memory capacity in bits or bytes, it is still common to use the decimal prefixes "kilo" (with symbol "k" or "K"), "mega" ("M"), "giga" ("G"), etc, to mean the closest binary prefixes "kibi", "mebi", etc. **This use is now deprecated by major standards bodies such as the [NIST](https://en.wikipedia.org/wiki/NIST) and the [BIPM](https://en.wikipedia.org/wiki/BIPM).**

</p>
</details>

**Comment by laurmaedje** [comment]
For reference, the last discussion about large WASM was here: https://github.com/typst/packages/pull/143.

One thing that would interest me is how performant is is. Is it huge, but usable performance-wise or is it also very slow?

**Comment by peng1999** [comment]
It's slow compared to [`jogs`](https://github.com/typst/packages/blob/main/packages/preview/jogs/0.2.1/README.md) but way faster than [`syntastica`](https://github.com/typst/packages/pull/143).

I tried this code as a basic benchmark:

```python
num_iterations = 10000
sum = "1"
# Perform the loop
for i in range(num_iterations):
    sum = str((int(sum) * i + 1) % 19260817)
```

It takes 1.549s on my machine. The similar code running by jogs spend 411.3 ms. The recalculation time shown by `typst watch` is 1.24s, and after I simply patch (see [diff](https://github.com/peng1999/typst/commit/f8744bff68894401a10d456a22291267d63605bf)) typst to use wasmtime instead of wasmi, the recalculation time is lower down to 60ms.

Machine info: 12th Gen Intel(R) Core(TM) i7-12700

**Review by laurmaedje — COMMENTED** [review]
There is no LICENSE file and no license is linked in the README. Since this is based on RustPython, there would ideally also be some form of attribution.

Is it a correct assessment that basing this on RustPython makes it slower, but smaller as opposed to the other mentioned distributions? Or are there other factors at play?

File: packages/preview/pyrunner/0.1.0/pyrunner.typ [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+// Usage:
+//
+// #let code = ```
+// def fib(n):
+//   return 1 if n <= 2 else fib(n-1) + fib(n-2)
+//
+// fib(5)
+// ```
+//
+// #python(code)
```
[the remark this answers is not in the document]
Comment (peng1999) [reply packages/preview/pyrunner/0.1.0/pyrunner.typ]: Oh that should be obsolete. Removed.

**Comment by laurmaedje** [comment]
I see. Thank you!

**Pull request merged**